paddle/framework/tensor.h

```diff
@@ -164,6 +170,80 @@ class Tensor {
   size_t offset_;
 };
 
+/*
+ * LODTensor
+ */
+class LODTensor {
+ public:
+// level_t save offsets of each unit.
+#ifdef PADDLE_ONLY_CPU
+  using level_t = std::vector<uint32_t>;
+#else
+  using level_t = thrust::device_vector<uint32_t>;
+#endif
+  // LOD stores offsets of each level of units, the largest units level first,
+  // then the smaller units level. Each level_t stores the offsets of units in
+  // Tesor.
+  typedef std::vector<level_t> lod_t;
+
+  explicit LODTensor() {}
+
+  LODTensor(LODTensor&& other)
```

Comment:
Why should we move an `LODTensor`? 

Maybe `LODTensor(LODTensor&& ) = delete` is enough?